3_visualize/src/4_save.R

```diff
@@ -0,0 +1,23 @@
+# Save the processed data
+save_processed_data <- function(data){
```

Comment:
Same advice applies here about including the `out_file` as an argument rather than a line at the end of a function.